File: src/server/MovieTheater.Business/ViewModels/Showtime/ShowtimeViewModel.cs
namespace MovieTheater.Business.ViewModels.Showtime;

public class ShowtimeViewModel : MasterBaseViewModel
{
    public Guid MovieId {get; set;}
    public Guid RoomId {get; set;}
    public DateOnly ShowDate {get; set;}
    public TimeSpan StartTime { get; set; }
    public string? MovieName {get; set;}
    public string? RoomName {get; set;}
}

File: src/client/src/models/showtime/showtimeview.model.ts
import { MasterBaseModel } from "../../core/models/master-base.model";

export class ShowtimeviewModel extends MasterBaseModel{
    public movieId!: string;
    public roomId!: string;
    public showDate!: Date;
    public startTime!: Date;
    public movieName!: string;
    public roomName!: string;
}
